Podfile

```diff
@@ -31,12 +31,6 @@ target 'VLC-iOS' do
   iOS_pods
```

Comment:
Due to the fact that `iOS_pods` are now only used by this single target, I think having those pods defined in a external function is not longer needed and they could be all moved to the `VLC-iOS` target itself